modules/apps/frontend-taglib/frontend-taglib-clay/src/main/resources/META-INF/resources/management_toolbar/ManagementToolbar.js

```diff
@@ -168,15 +169,40 @@ function ManagementToolbar({
 									<ClayDropDownWithItems
 										items={viewTypeItems}
 										trigger={
-											<ClayButtonWithIcon
-												className="nav-link nav-link-monospaced"
-												displayType="unstyled"
-												symbol={
-													viewTypeItems.find(
-														(item) => item.active
-													)?.icon || ''
-												}
-											/>
+											showDesignImprovementsFF ? (
+												<ClayButton
+													className="nav-link"
+													displayType="unstyled"
+													title={Liferay.Language.get(
+														'show-view-options'
+													)}
+												>
+													<ClayIcon
+														symbol={
+															viewTypeItems.find(
+																(item) =>
+																	item.active
+															)?.icon || ''
```

Comment:
Why do we pass an empty string as a symbol? If there's no active item we shouldn't render any icon at all, right?

Anyway, is there any use case where it's possible to not having an active view? Also, if we do this finding here we'll be iterating over the entire array on every render. Would have sense to memoize this active item?

Reply:
> If there's no active item we shouldn't render any icon at all, right?

Yup, that's the end result. `ClayButtonWithIcon` will display no icon both if you pass an empty string or nothing/undefined. The only difference is that if you pass an empty string it renders the `lexicon-icon-` class instead of `lexicon-icon-undefined`, which yields the same result. I personally prefer the first (undefined in the class gives me an eye twitch), but I don't feel too strongly about it. BTW, in this case I just copy-pasted what we were doing before this PR.

> is there any use case where it's possible not having an active view

I don't think there should be an use case for this. If you're enabling this button by passing the `viewTypeItems` prop, that should always have an active view, and it should be handled by the specific product team's code.

>Would have sense to memoize this active item? 

Since `viewTypeItems` is probably a small array (haven't seen more than 3 views so far) I don't see how it will have that much impact, but I'll add it if you think it's not the case!

Reply:
As we talked offline we can get rid of the `||` logic